### DISCUSSION TITLE
Release 2.0.0a3 - alpha quality, for testing only

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#6833] [General]
borgbackup 2.0.0a3 alpha release - do not use this on production backup repositories.

Long changelog:
https://borgbackup.readthedocs.io/en/2.0.0a3/changes.html#version-2-0-0a3-2022-07-04

If you use pip to install this, use: pip install "borgbackup==2.0.0a3"

Docs: https://borgbackup.readthedocs.io/en/2.0.0a3/

Pyinstaller-made fat binaries coming soon!

Feedback: here below or in the issue tracker.

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/200a3'>Release 2.0.0a3 - alpha quality, for testing only</a>.</em>

**Comment by ThomasWaldmann** [comment]
Please note that `borg transfer` has new options and you'll need to choose the right one depending on whether you want:
- to transfer archives between 2 related borg2 repos (this is the default: it will not apply any changes / conversions / upgrades to the data)
- to transfer archives from a borg 1.2 repo to a related borg 2.0 repo (needs option `--upgrader=From12To20`)

Please test archive transfers to help avoiding that bugs in there will make it into the 2.0 release.

Of special interest:
- hardlinks
- xattrs, acls
- (bsd)flags
- repos that exist since long (and were written to by early borg or attic releases)

Please always use **copies** of you production repos to do experiments with alpha releases.

**Comment by ThomasWaldmann** [comment]
The borg2 stuff still lives in a separate `borg2` branch right now.

If there is no negative feedback from testers, I'll merge it into master soon.

**Comment by ThomasWaldmann** [comment]
Did anyone try alpha3? Please post your results below. :-)

BTW, if you post feedback, please tell which pyinstaller-made binary you used or whether you used pip or installed from git.

**Comment by real-yfprojects** [comment]
Will v2 support v1 repositories for write actions?

**Reply by infectormp** [reply]
No, you can reed more about v2 here https://github.com/borgbackup/borg/issues/6602

**Reply by ThomasWaldmann** [reply]
v1 repo --- borg transfer ---> v2 repo.

**Reply by real-yfprojects** [reply]
> No, you can reed more about v2 here #6602

Thanks. I didn't know about that post yet.
